### PR TITLE
Update uninstallation instructions

### LINKED ISSUE #559: Update uninstall process doc
There have been a number of questions related to failed uninstalls after attempting to uninstall by deleting `cert-manager` namespace see [cert-manager#3986](https://github.com/jetstack/cert-manager/issues/3986), [cert-manager#3474](https://github.com/jetstack/cert-manager/issues/3474) [cert-manager#3870](https://github.com/jetstack/cert-manager/issues/3870).

We already have some notes on how to recover from this scenario [here](https://cert-manager.io/docs/installation/uninstall/kubernetes/#namespace-stuck-in-terminating-state), but we could update that section with how to manually delete pending custom resources (as mentioned in the above linked issues).

### PR DESCRIPTION
**Pull request by enjeck**
Closes #559

### CODE REVIEW

**Comment by enjeck** [comment]
@irbekrm I've made changes :)

File: content/docs/installation/kubectl.md [hunk 1]
```diff
@@ -67,8 +67,9 @@ cert-manager using the procedure determined by how you installed.
 
 > **Warning**: Uninstalling cert-manager or simply deleting a `Certificate` resource can result in 
 > TLS `Secret`s being deleted if they have `metadata.ownerReferences` set by cert-manager.
-> You can disable this behavior by setting `-enable-certificate-owner-ref=false` before deleting
-> a `Certificate` or `cert-manager`.
+> By default in cert-manager 1.8+, `--enable-certificate-owner-ref` is set to `false`, which will ensure
+> that owner references are removed.
```
Comment (irbekrm) [inline content/docs/installation/kubectl.md:71]: Suggestion:

You can control the whether owner references are added to `Secret`s using `--enable-certificate-owner-ref` controller flag. By default this flag is set to false, which means that no owner references are added. However, in cert-manager older than v1.8 changing the flag's value from true to false _did not_ result in existing owner references being removed. This behaviour was fixed in cert-manager v1.8.

**Review by irbekrm — CHANGES_REQUESTED** [review]
Thanks for making the changes @enjeck !

This looks good to me!

I've added a wording suggestion around the owner refs- that part is a little complex, so I'd like to make sure that users understand how it works to minimize the chance of someone accidentally deleting their certs that could result in application downtime and/or unnecessary re-issuances.
Let me know if that makes sense.

**Comment by irbekrm** [comment]
Thanks for working on this @enjeck 

I think the doc looks a lot better now!

/ok-to-test
/lgtm
/approve

**Comment by maelvls** [comment]
We paired on the failing build this morning, and weren't able to spot where the spelling mistake is. The error is:

```
 +++ Running spell check
    content/docs/installation/kubectl.md
       73 | eferences being removed. This behaviour was fixed in cert-manager v1. 
       73 | our was fixed in cert-manager v1.8. 
>> 2 spelling errors found in 1152 files 
```

I'll try to run the spell checker locally by running

```
./script/verify
```

**Comment by maelvls** [comment]
I was able to run `./scripts/verify` locally, and it showed the spelling issues in red:

Screenshot from 2022-05-02 12-17-55

It seems like `v1.8` is already in `.spelling`, but the spell checker behaves differently when `v1.18` ends with a dot. It doesn't understanding end-of-sentence periods... 😞

**Workaround proposition:** add `v1.8.` to the .spelling file. 😇

> Note: I was struggling during our call due to #957. I just had to run `git cleanup -fd` to fix the "hanging build".

**Comment by enjeck** [comment]
@maelvls Thank you!

**Comment by enjeck** [comment]
@maelvls Looks good now. Thank you!

**Pull request merged**